### ISSUE TITLE
Add link in UI to docs on how to add another cluster

### ISSUE THREAD

**Issue by soupdiver** [argoproj/argo-cd#14170]
# Summary

We can remove a cluster via the UI but not add a new one. Adding a cluster would also be nice.

# Motivation

Avoiding using CLI within argocd running in a cluster.

# Proposal

Add a form to add a cluster in the web ui.

This was already asked in #3551 but in the follow up PR only removal of a cluster was added

**Comment by ishitasequeira** [comment]
This enhancement was brought in the contributors call today. Currently, the `argocd` cli reads the user's kube-config to create secrets and service accounts for adding the cluster to Argo CD. If we have to do the same via UI, users would need to input a lot more information and we would need to add a lot more validations. Thus, during the contributor's call, everyone came to a consensus on not having cluster addition via UI but instead add example on how to add clusters using or point to the documentation from the UI.

**Comment by soupdiver** [comment]
> This enhancement was brought in the contributors call today. Currently, the `argocd` cli reads the user's kube-config to create secrets and service accounts for adding the cluster to Argo CD. If we have to do the same via UI, users would need to input a lot more information and we would need to add a lot more validations. Thus, during the contributor's call, everyone came to a consensus on not having cluster addition via UI but instead add example on how to add clusters using or point to the documentation from the UI.

I agree, this totally makes sense and totally haven't thought about this aspect.
I guess the part that involves reading the users kube-config is what is hard/impossible to do nicely through the web ui.
Thanks for clarification ❤️

**Comment by crenshaw-dev** [comment]
I'll re-title and leave this open so we can track the UI-link improvement.  :-)

**Comment by shivamb** [comment]
@ishitasequeira 

We can just upload the config file from UI. All details does not need to be captured in separate fields.
